corehq/apps/commtrack/sms.py

```diff
@@ -124,27 +123,13 @@ def parse(self, text):
             args = args[1:]
 
         action = self.commtrack_settings.action_by_keyword(action_keyword)
-        if action and action.type == 'stock':
```

Comment:
this "type" property isn't actually a declared property of the `CommtrackActionConfig` document schema class, but rather something that was being hacked in by `CommtrackConfig.action_by_keyword` to differentiate stock actions from requisition actions, which exist separately in the config but which this method returns together in a single dict. Since there's now only one type (stock), there's no need to hack in the type property. If you look for `action_by_keyword` in the PR diff, you'll see I also removed the part of that method that hacks `type` in.